README.md

```diff
@@ -107,6 +106,103 @@ print(myShoppingList)
 // prints "["Almond Butter", "Jelly", "Bread", "Strawberries", "Bananas"]"
 ```
 
-* Challenge the student to write a function that takes in an `Array` as an argument, the `Array`'s type is `[String]`, where they are asked to grab the first element in the array (at index 0) and see if it's value is equal to "Bread". If so, return `true`.. if not, return `false`.
+- Challenge the student to write a function that takes in an `Array` as an argument, the `Array`'s type is `[String]`, where they are asked to grab the first element in the array (at index 0) and see if it's value is equal to "Bread". If so, return `true`.. if not, return `false`.
+
+--------------------------------------------------------------------------------
+
+## Arrays
+
+So far you have learnt about several types of variable to represent numbers, strings, and other values. This lesson will introduce a new type of variable, one that allows it to contain a _collection_ of values of the same type. These are called _arrays_.
+
+Arrays are not the only collection type available in Swift and you will cover others later in the course, but for now it's important to know that an array is an _ordered_ collection of values, i.e. that every value has a particular place (or _index_) in the collection.
+
+For example, take this (conceptual, in no particular programming language) array of items on a shopping list for a sandwich:
+
+```
+0 => "Bread",
+1 => "Butter",
+2 => "Cheese",
+3 => "Lettuce",
+4 => "Tomatoes",
+```
+
+This is a collection of five string values. The item on the right hand side of the `=>` symbol is a string value, and the number to the left it's position in the array. Note that an array starts from 0, and not 1\. This is common in programming languages, so for a five item array, the last item will actually be at position number 4.
+
+Whilst arrays share similar concepts and principles across programming languages, you create and use them differently in different languages.
+
+## Creating an Array
+
+Here's how to create an array of values in Swift:
+
+```swift
+var list = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+In this case, you are creating a variable with the `var` keyword called 'list'. Swift will decide on the type of values allowed in the array for you as you add them.
+
+To be more precise, you can declare the type of values allowed in the array explicitly:
+
+```swift
+var shoppingList: [String] = ["Bread", "Butter", "Cheese", "Lettuce", "Tomatoes"]
+```
+
+Here you are creating a variable with the `var` keyword called 'shoppingList', setting the type of values allowed explicitly to be of `String` and then adding five string values to the array.
+
```

Comment:
This is great! I love being very explicit with a sentence like this following a piece of code because it challenges the student to see if they really know what's going on. As well, they can become articulate on how to describe something which helps them learn and helps them google things later. 

👍